client/my-sites/stats/controller.jsx

```diff
@@ -167,7 +167,34 @@ export function redirectToDefaultModulePage( context ) {
 }
 
 export function insights( context, next ) {
-	context.primary = <StatsInsights />;
+	const {
```

Comment:
This has to be simplified. More details [here](https://github.com/Automattic/wp-calypso/pull/71159/files#r1051280601).